plugins/org.eclipse.osee.framework.skynet.core/src/org/eclipse/osee/framework/skynet/core/artifact/RelationLoader.java

```diff
@@ -93,7 +108,20 @@ public static void loadRelationData(Long joinQueryId, Collection<Artifact> artif
       }
 
       for (Artifact artifact : artifacts) {
+         setTransactionRecord(artToTx, artifact);
          artifact.setLinksLoaded(true);
       }
    }
+
+   private static void setTransactionRecord(Map<ArtifactToken, TransactionToken> artToTx, Artifact artifact) {
+      TransactionToken txTok = artToTx.get(artifact);
+      if (txTok != null && txTok.isValid()) {
```

Comment:
This should never be null if it is coming from the DB since me and Audrey made the change to prevent strings being nulled improperly.

Reply:
This is a cache so don't have to load the TransactionRecord multiple times per artifact.